zerver/lib/push_notifications.py

```diff
@@ -842,8 +849,14 @@ def get_apns_alert_subtitle(
             )
         else:
             return _("{full_name} mentioned you:").format(full_name=message.sender.full_name)
+    elif trigger == NotificationTriggers.TOPIC_WILDCARD_MENTION_IN_FOLLOWED_TOPIC:
+        return _("TODO - 2")
     elif trigger == NotificationTriggers.STREAM_WILDCARD_MENTION_IN_FOLLOWED_TOPIC:
         return _("TODO")
+    elif trigger == NotificationTriggers.TOPIC_WILDCARD_MENTION:
+        return _("{full_name} mentioned all the topic participants:").format(
+            full_name=message.sender.full_name
```

Comment:
maybe "mentioned all topic participants:" would be better -- I don't think we need the " the " here.

Reply:
Done.

Also removed "the" at line 686 in this file.

Reply:
Done.

Also removed "the" at line 686 in this file.